algopy_testing/src/algopy/primitives/bytes.py

```diff
@@ -80,6 +75,24 @@ def __getitem__(
     def __eq__(self, other: object) -> bool:
         return self.value == _as_bytes(other)
 
+    def __and__(self, other: bytes | Bytes) -> Bytes:
+        return self._operate_bitwise(other, "and_")
+
+    def __rand__(self, other: bytes) -> Bytes:
+        return self & other
+
+    def __or__(self, other: bytes | Bytes) -> Bytes:
+        return self._operate_bitwise(other, "or_")
+
+    def __ror__(self, other: bytes) -> Bytes:
+        return self | other
```

Comment:
@daniel-makerx should there be a check on whether other is indeed of type bytes?

Reply:
This should be fine, as these bitwise operations are all commutative, so we can just forward the implementation on to the reflected operations which do the type check